using_images/s2i_images/dot_net_core.adoc

```diff
@@ -0,0 +1,85 @@
+[[using-images-using-dot-net-core]]
+= .NET Core
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[benefits-of-using-dot-net-core]]
+== Benefits of Using .NET Core
+
+{nbsp}.NET Core is a general purpose development platform featuring automatic memory
+management and modern programming languages. It allows users to build high
+quality applications efficiently. .NET Core is available on Red Hat Enterprise
+Linux (RHEL 7) and {product-title} via certified containers. .NET Core offers:
+
+* The ability to follow a microservices-based approach, where some components are
+built with .NET and others with Java, but all can run on a common, supported
+platform in Red Hat Enterprise Linux and {product-title}.
+* The capacity to more easily develop new .NET Core workloads on Windows;
+customers are able to deploy and run on either Red Hat Enterprise Linux or
+Windows Server.
+* A heterogeneous data center, where the underlying infrastructure is capable of
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+{nbsp}.NET Core can produce a multi-architecture binary, including compiled user
+code and the .NET virtual machine (VM). This can be used to build a smaller VM
+for deployment.
+
+[[dot-net-core-supported-versions]]
+== Supported Versions
+
+* .NET Core framework version 1.0.
+* .NET Core is supported by Red Hat Enterprise Linux (RHEL) 7.
+* {product-title} versions 3.3 and later are supported.
+
+[[dot-net-core-installing-images]]
+== Images
+
+Image stream definitions for the .NET Core on RHEL S2I image are now added
+during {product-title} installations.
+
+[[dot-net-core-configuration]]
+== Configuration
+
+The .NET Core image currently does not support any environment variables. No
+configuration is necessary.
+
+[[dot-net-compile-sources]]
+== Compile Sources
+
+You must compile the sources before you can run your application.
+
+. Restore the dependencies and tools of a project:
++
+----
+$ dotnet restore
+----
+
+. Build the project and all of its dependencies:
++
+----
+$ dotnet build
+----
+
+[[dot-net-core-run-your-application]]
+== Run Your Application
+
+To run your application:
+
+----
+$ dotnet run
+----
+
+[NOTE]
+====
```

Comment:
@bparees @jerboaa Added this as a note for now, but it would be nice to include a full example. Do we have a standalone sample repo?